core/build.gradle.kts

```
@@ -63,6 +63,7 @@ kotlin {
                 // Workaround for AtomicFU plugin not automatically adding JVM dependency for Android.
                 // https://github.com/Kotlin/kotlinx-atomicfu/issues/145
                 implementation(libs.atomicfu)
+                implementation(libs.tuulbox.coroutines)
```

nit: newline between lines 65 and 66, to help clearly indicate that comment is only for line 65, and doesn't apply to this dependency as well

```suggestion
                // Workaround for AtomicFU plugin not automatically adding JVM dependency for Android.
                // https://github.com/Kotlin/kotlinx-atomicfu/issues/145
                implementation(libs.atomicfu)

                implementation(libs.tuulbox.coroutines)
```